resources/configurationFiles/analysisRNAseq.xml

```diff
@@ -63,13 +67,15 @@
         <cvalue name='STAR_VERSION' value='2.5.3a' type='string'/>
         <cvalue name='SUBREAD_VERSION' value='1.5.1' type='string'/>
         <cvalue name='SAMBAMBA_VERSION' value='0.6.5' type='string'/>
-        <cvalue name='SAMTOOLS_VERSION' value='1.6' type='string'/>
-        <cvalue name='HTSLIB_VERSION' value='1.6' type='string'/>
+        <cvalue name='SAMTOOLS_VERSION' value='1.9' type='string'/>
+        <cvalue name='HTSLIB_VERSION' value='1.9' type='string'/>
         <cvalue name='RNASEQC_VERSION' value='1.1.8' type='string'/>
         <cvalue name='KALLISTO_VERSION' value='0.43.0' type='string'/>
         <cvalue name='QUALIMAP_VERSION' value='2.2.1' type='string'/>
-        <cvalue name='ARRIBA_VERSION' value='1.2.0' type='string'/>
+        <!--Update to Arriba version 2+ for both hg19 and hg38 genome reference-->
```

Comment:
I agree, the major version bump will help the users to distinguish between the Arriba versions.

Reply:
With a stand-alone Arriba workflow on the horizon, we should consider disabling Arriba altogether. It could cause confusion on the end-user's side when output files are available at multiple locations and especially if the Arriba versions get out of sync. There were good reasons to couple alignment and fusion detection in the past. But the viral detection feature of the new Arriba version requires a separate alignment, so these reasons are no longer valid. @vinjana: I will get in touch with you shortly regarding updates to my stand-alone Arriba workflow. We can discuss disabling Arriba in the RNA-Seq workflow on that occasion.

Reply:
I agree multiple results variation would confusion the end user. The plan was to make the Arriba runnable and based on the consensus we could disable it from default runs. In that case, we could also revert to Arriba version 1.2 for the h19, so we do not need a major version bump.

Reply:
I agree as well.